test/suite/test_prepare04.py

```diff
@@ -106,14 +106,17 @@ def test_prepare_conflict(self):
         s_other.begin_transaction(self.txn_config + self.ignore_config)
         c_other.set_key(1)
         if self.ignore == False and self.after_ts == True:
-            self.assertRaises(wiredtiger.WiredTigerError, lambda:c_other.search())
+            # WT_PREPARE_CONFLICT error returned when a read encounters a prepared transaction update
+            self.assertRaisesException(wiredtiger.WiredTigerError, lambda:c_other.search(), preparemsg)
         else:
             c_other.search()
             self.assertTrue(c_other.get_value() == 1)
+
         c_other.set_value(3)
-        self.assertRaises(wiredtiger.WiredTigerError, lambda:c_other.update())
+
+        # WT_ROLLBACK error returned due to conflict with concurrent operations
+        self.assertRaisesException(wiredtiger.WiredTigerError, lambda:c_other.update(), conflictmsg)
```

Comment:
Similar comment here.
`# Make sure we detect the conflict between operations.`

Reply:
updated according to your comment.